xml/form.xsd

```diff
@@ -50,6 +56,19 @@
 			</xsd:sequence>
    </xsd:complexType>
 
+	<xsd:complexType name="comboBoxType">
```

Comment:
sry falsch gemergt, korrigiere ich gleich. Gab es schon, ich habe nur etwas abgeändert wegen eigener option-klasse. Die Objektstruktur wird mit dem zweiten Commit in diesem Pull-Request übernommen.